jdi-light-angular/src/main/java/com/epam/jdi/light/angular/asserts/ButtonToggleAssert.java

```diff
@@ -2,7 +2,34 @@
 
 import com.epam.jdi.light.angular.elements.complex.ButtonToggle;
 import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import org.hamcrest.Matchers;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
 
 public class ButtonToggleAssert extends UIAssert<ButtonToggleAssert, ButtonToggle> {
 
+    @JDIAction("Assert that '{name}' has '{0}' class")
+    public ButtonToggleAssert assertButtonToggleIsSelected(String value) {
```

Comment:
Assertion method name is not reasonable. From my point of view "selected" will be fairly well here.